README.md

```diff
@@ -49,7 +49,9 @@ Stampit uses [three different kinds of prototypal OO](https://vimeo.com/69255635
 [![NPM](https://nodei.co/npm/stampit.png)](https://www.npmjs.com/package/stampit)
 
 Browsers: [![CDNJS](https://img.shields.io/badge/cdnjs.com--green.svg)](https://cdnjs.com/libraries/stampit)
-[![UNPKG](https://img.shields.io/badge/unpkg.com--green.svg)](https://unpkg.com/stampit)
+[![UNPKG](https://img.shields.io/badge/unpkg.com--green.svg)](https://unpkg.com/stampit@latest/dist/)
+
+**WARNING!** If running in node.js <= v0.12 or IE <= 11 then you'd need to polyfill the `Object.assign`. Like [this](https://github.com/sindresorhus/object-assign), or [that](https://github.com/Financial-Times/polyfill-service/blob/master/polyfills/Object/assign/polyfill.js), or [else](https://npms.io/search?q=keywords%3Aponyfill).
```

Comment:
I'm lost. Could you please provide what you wanna see here? I don't understand. :)
Just push to this branch. Or edit on the GitHub. Whatever is easier for you. :)
Thanks!